src/clusterfuzz/_internal/metrics/logs.py

```diff
@@ -687,12 +687,12 @@ def emit(level, message, exc_info=None, **extras):
 
   path_name, line_number, method_name = get_source_location()
 
-  # Don't pass exc_info at all, as otherwise cloud logging will append
```

Comment:
@hunsche Why was this change made in the first place?

Reply:
I've explained the reasoning for this change in the comment thread here: https://github.com/google/clusterfuzz/pull/4868#discussion_r2211347867